src/stream/src/executor/actor.rs

```diff
@@ -164,9 +164,17 @@ where
 
         let span_name = format!("Actor {id}");
 
-        let new_span =
-            || tracing::info_span!(parent: None, "actor", "otel.name" = span_name, actor_id = id);
-        let mut span = new_span();
+        let new_span = |epoch: Option<EpochPair>| {
+            tracing::info_span!(
+                parent: None,
+                "actor",
+                "otel.name" = span_name,
```

Comment:
What's otel?

Reply:
It's a special field for opentelemetry. Will be displayed as the span name.

https://docs.rs/tracing-opentelemetry/latest/tracing_opentelemetry/#special-fields